checkov/kubernetes/checks/resource/k8s/KubeletClientCa.py

```diff
@@ -15,14 +15,12 @@ def scan_container_conf(self, metadata: Dict[str, Any], conf: Dict[str, Any]) ->
         self.evaluated_container_keys = ["command"]
         if conf.get("command"):
             if "kubelet" in conf["command"]:
+                hasClientCaFile = False
                 for command in conf["command"]:
-                    if command.startswith("--root-ca-file"):
-                        file_name = command.split("=")[1]
-                        extension = file_name.split(".")[1]
-                        if extension == "pem":
-                            return CheckResult.PASSED
-                        else:
-                            return CheckResult.FAILED
+                    if command.startswith("--client-ca-file"):
+                        if len(command.split("=")) == 2:
+                            hasClientCaFile = True
+                return CheckResult.PASSED if hasClientCaFile else CheckResult.FAILED             
```

Comment:
```suggestion
                    if command.startswith("--client-ca-file"):
                        if len(command.split("=")) == 2:
                            return CheckResult.PASSED
                return CheckResult.FAILED             
```
you can just remove `hasClientCaFile` completely 🙂

Reply:
Removed the variable and also I optimised the code for a better check, and I believe this is how it should be in other checks as well.

Reply:
there is a high chance that the other checks are a bit outdated 😄